k4LCIOReader/src/k4LCIOConverter.cc

```diff
@@ -1101,7 +1101,7 @@ podio::CollectionBase *k4LCIOConverter::cnvAssociationCollection(EVENT::LCCollec
         {
             auto rval = (EVENT::LCRelation *)src->getElementAt(i);
             if((EVENT::ReconstructedParticle *)rval->getFrom()==0 || (EVENT::Vertex *)rval->getTo()==0) continue;//remove 0
-            edm4hep::RecoParticleVertexAssociation lval = dest->create();
+            edm4hep::MutableRecoParticleVertexAssociation lval = dest->create();
```

Comment:
Hi Placido @fdplacido ,
Should we use "auto" instead?
Tao

Reply:
totally, thanks for pointing that out! :)
Placido